worker-workflow/src/main/resources/workflow-control.js

```diff
@@ -184,6 +184,16 @@ function fieldExists(document, fieldName) {
     return document.getField(fieldName).hasValues();
 }
 
+function isNotEmptyMap(obj) {   
+    if(!obj)
+        return false;
```

Comment:
This returns `false` (i.e. that the parameter is an empty object) in quite a lot of cases where the parameter is not an empty object, such as if it was an empty string or null or not present.  Perhaps that is intentional?